ticdc/ticdc-overview.md

```diff
@@ -54,7 +54,8 @@ TiCDC 的系统架构如下图所示：
 
 - MySQL sink
 
-    - TiCDC 不拆分表内事务，**保证**单表事务一致性，但**不保证**与上游表内事务的顺序一致。如果事务间的主键有交集，依然可以保证与上游顺序一致。
+    - TiCDC 不拆分单表事务，**保证**单表事务原子性。
+    - TiCDC **保证**单表事务顺序一致性，**注意** v4.0.5 之前版本没有主键或唯一索引的表**不保证**顺序一致性。
```

Comment:
https://github.com/pingcap/ticdc/pull/843 doesn't change the result of causality detection, so the delivery guarantee in MySQL sink is not changed.
Have you met unexpected event sequence in test?

Reply:
I will modify this description later